networking/kvm.go

```diff
@@ -569,6 +569,13 @@ extend Networking struct with methods to clean up kvm specific network configura
 // removing tuntap interface and releasing its ip from IPAM plugin
 func (n *Networking) teardownKvmNets() {
 	for _, an := range n.nets {
+		if an.conf.Type == "flannel" {
```

Comment:
@jonboulle I can't. At first, I transform flannel network and then pass to the switch statement. We use similar check here:
https://github.com/coreos/rkt/blob/master/networking/kvm.go#L431